packages/extension-chakra-storefront/src/components/with-layout/with-layout.tsx

```diff
@@ -229,8 +220,8 @@ const withLayout = <P extends object>(WrappedComponent: React.ComponentType<P>)
                 </Helmet>
 
                 <Seo>
-                    <meta name="theme-color" content={THEME_COLOR} />
-                    <meta name="apple-mobile-web-app-title" content={DEFAULT_SITE_TITLE} />
+                    <meta name="theme-color" content={colors.blue['600']} />
```

Comment:
THEME_COLOR is a part of colors in theme, we can reuse it instead of creating a new const